proxy.go

```diff
@@ -239,7 +239,18 @@ func printAgentLogs(sock string) error {
 	otherMask := 0007
 	other := int(fileInfo.Mode().Perm()) & otherMask
 	if other != 0 {
-		return fmt.Errorf("All socket permissions for 'other' should be disabled, got %3.3o", other)
+		logger().WithField("socket permissions", fmt.Sprintf("%4.4o", fileInfo.Mode().Perm())).
+			Infof("All socket permissions for 'other' should be disabled")
```

Comment:
Nit: This can now be `Info()` as you don't have any formats to expand in the string.

Reply:
done.